suite-common/graph/src/graphUtils.ts

```diff
@@ -33,8 +36,7 @@ export const getTimestampsInTimeFrame = (
         numberOfPoints,
     });
 
-    // sometimes endOfRangeDate could be too close to recent time and that cause problems with blockbook
-    const intervalEndDate = fromUnixTime(getBlockbookSafeTime(getUnixTime(endOfTimeFrameDate)));
+    const intervalEndDate = fromUnixTime(getUnixTime(endOfTimeFrameDate));
```

Comment:
How have you found out that `getBlockbookSafeTime` is no longer necessary?

Reply:
I think from @martinboehm quite long time ago